doc/show.md

```diff
@@ -0,0 +1,32 @@
+# show command (Winget-Create)
+
+The **show** command of the [Winget-Create](../README.md) tool is designed to show an existing manifest from the [Windows Package Manager repo](https://docs.microsoft.com/windows/package-manager/). This command is particularly useful when you need to update an existing package and wish to examine an older manifest without leaving the CLI. This helps you determine the count of existing installer nodes and decide if architecture or scope overrides are necessary for the update.
+
+## Usage
+
+Show the latest manifest of an existing package from the Windows Package Manager repo:
+
+`wingetcreate.exe show <PackageIdentifier> --token <GitHubPersonalAccessToken>`
+
+Show a specified version of a package manifest:
+
+`wingetcreate.exe show <PackageIdentifier> --version <PackageVersion> --token <GitHubPersonalAccessToken>`
+
+Show only the installer and default locale manifest:
+
+`wingetcreate.exe show <PackageIdentifier> --installer-manifest --defaultlocale-manifest --token <GitHubPersonalAccessToken>`
+
+## Arguments
+
+The following arguments are available:
+
+| Argument  | Description |
+|--------------|-------------|
+| **-v, --version** |  The version of the existing package manifest from the Windows Package Manager Repo to retrieve the manifest for. Default is the latest version.
+| **--installer-manifest** |  Switch to display the installer manifest.
+| **--defaultlocale-manifest** |  Switch to display the default locale manifest.
+| **--locale-manifests** |  Switch to display all locale manifests.
+| **--version-manifests** |  Switch to display the version manifest.
```

Comment:
nit: I feel that we need to add a short name argument for some of these i'm not sure what...

Maybe something like:
-i, --installer-manifest
-d --defaultlocale-manifest
-l, --locale-manifests
--version-manifest

I am aware of the conflict between --version and --version-manifest so I am okay with leaving out a short form since I don't think this will be a common arg.

Reply:
Added short name arguments